### PR TITLE
Aspect-fill images automatically

### LINKED ISSUE #49: Add aspect fill / fit option
When displaying an image, Plash resizes to fit the image within the boundaries of the screen while keeping the aspect ratio. If the source image isn't the same aspect ratio as the screen (minus the toolbar height), the left and right sides will be cropped in and you can see the original desktop underneath.

My request is that an aspect fill option is added. Plash could then be an arbitrary size and the image would be guaranteed to fill the whole thing.

### PR DESCRIPTION
**Pull request by colejd**
Resolves #49.

When an image is loaded, it will stretch to aspect-fill the screen.

### CODE REVIEW

**Comment by sindresorhus** [comment]
We need to rename the method.

Can you include a code comment for a URL that can be used for testing it?

**Comment by colejd** [comment]
Fixed.

Here's an image I used to test on a 4k monitor:
desktoptest

**Comment by sindresorhus** [comment]
Could you put the first and last in a Codepen/JSFiddle and add code comment with links, so it can easily be tested for someone needing to touch the code in the future?

**Comment by sindresorhus** [comment]
I think we should also mention the behavior in the readme. In one sentence.

**Comment by colejd** [comment]
Here's the Codepen: https://codepen.io/colejd/pen/MWyxezd

I'll add it to the code in just a moment.

File: Plash/Utilities.swift [hunk 1]
```diff
@@ -1679,19 +1679,32 @@ extension WKWebView {
 	// TODO: Use https://developer.apple.com/documentation/webkit/wkwebview/3516410-mediatype when macOS 10.15.4 is out.
 	// TODO: Move this to `SSWebView` instead and also expose a `response` property so we don't need the `mimeType` parameter.
 	// https://github.com/feedback-assistant/reports/issues/82
-	/// Center a standalone image as WKWebView doesn't center it like Chrome and Firefox do...
-	func centerImage(mimeType: String?) {
+	/**
+	Centers a standalone image as WKWebView doesn't center it like Chrome and Firefox do.
+	The image will aspect-fill the space available.
+
+	You can test this functionality at https://codepen.io/colejd/pen/MWyxezd
```
Comment (sindresorhus) [inline Plash/Utilities.swift:1686]: I think you misunderstood the request. I was asking for a URL that can be put into Plash to test that Plash does the correct thing. In hindsight, I guess we cannot use CodePen for this as it doesn't let us show the full page. Not important enough to spend more time on this.

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**